src/service_config/contrib_portal_config.py

```diff
@@ -14,3 +15,10 @@ def add_templates_to_contrib_portal(service, path_to_templates):
     print("- Copying burden estimate templates from orderly to contrib portal")
     copy_between_volumes(service.volume_name("orderly"), service.volume_name("templates"), path_to_templates)
 
+
+def get_template_report_versions():
+    with open("template_report_versions") as f:
+        value = f.read()
+    if len(value) == 0:
+        return []
+    return value.split()
```

Comment:
Consider 

```python
[x.strip() for x in value.split("\n") if x]
```

to go with newline delimited and to strip the trailing newline (and over any blank lines)

Reply:
That also means you can avoid the early exit